lib/process-services/src/lib/process-list/components/start-process.component.spec.ts

```diff
@@ -747,4 +709,104 @@ describe('StartFormComponent', () => {
             expect(processNameInput.disabled).toEqual(false);
         });
    });
+
+    describe('Empty Template', () => {
+
+        beforeEach(() => {
+            fixture.detectChanges();
+            getDefinitionsSpy.and.returnValue(of([]));
+            getDeployedApplicationsSpy = spyOn(appsProcessService, 'getDeployedApplications').and.returnValue(of([]));
+        });
+
+        it('should show no process definition available template when application/process definitions are empty', () => {
+            component.showSelectApplicationDropdown = true;
+            const change = new SimpleChange(null, 3, true);
+            component.ngOnChanges({ 'appId': change });
+            component.applications = [];
+            component.processDefinitions = [];
```

Comment:
You are spying on the getDeployedApplications and getDefinitions and returning empty array so the `component.applications` and the `component.processDefinitions` should already be empty right?

Reply:
Updated, Thanks @arditdomi